class.jetpack-gutenberg.php

```diff
@@ -89,6 +93,36 @@ class Jetpack_Gutenberg {
 	 */
 	private static $availability = array();
 
+	/**
+	 * Check to see if a block is able to run with current wp or gutenburg plugin versions
+	 *
+	 * @param array $version_requirements An arry containing the minimum 'wp' and 'plugin' versions for the block.
+	 *
+	 * @return boolean True if the block is gated due to current wp or plugin versions.
+	 */
+	public static function is_block_version_gated( $version_requirements ) {
+		global $wp_version;
+
+		if ( defined( 'IS_WPCOM' ) && IS_WPCOM ) {
+			return false;
+		}
+
+		if ( is_file( WP_PLUGIN_DIR . '/gutenberg/gutenberg.php' ) ) {
```

Comment:
This isn't the most robust way to check for plugin, as it could be in a different folder as it is on wpcom, but `GUTENBERG_VERSION` doesn't seem to be available in this context, so not sure that there is any other way.

Reply:
This is a reasonable argument, I'm fine with doing it this way.

Reply:
Could `GUTENBERG_VERSION` not be available because you're running a dev version? If so, [it only defines development mode](https://github.com/WordPress/gutenberg/blob/master/gutenberg.php#L13-L15), but with the production plugin, those defines get replaced with the git commit and the version number

Assuming that's all the case, then you could check for `GUTENBERG_DEVELOPMENT_MODE` is `true` or `GUTENBERG_VERSION` is above the required one

Reply:
Thanks @pablinos, the issue was the fact that I was checking for `GUTENBERG_VERSION` in dev environment. It is set in this context if running on sandbox, etc. @pento, what do you think about Paul's suggestion of checking for `GUTENBERG_DEVELOPMENT_MODE` or `GUTENBERG_VERSION`?

Reply:
Do you think it'd be possible to move this outside of this Google Calendar block PR?

It may make both the block and the new gating methods easier to review if they're split into 2 different PRs.

Reply:
That sounds sensible, especially if we explore this idea of adding a nudge to upgrade.

Reply:
>Do you think it'd be possible to move this outside of this Google Calendar block PR? 

Done.